lib/store/utxo.ex

```diff
@@ -8,6 +8,7 @@ defmodule Elixium.Store.Utxo do
 
   @store_dir "utxo"
   @ets_name :utxo
+  @keys "keys"
```

Comment:
I think this is unnecessary, we can just pass in the string "keys"

Reply:
Refactored & Fixed